arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -1774,7 +1776,9 @@ arangodb::Result IResearchAnalyzerFeature::remove( // remove analyzer
 }
 
 void IResearchAnalyzerFeature::start() {
-  ApplicationFeature::start();
+  if (!isEnabled()) {
```

Comment:
is it necessary to bail out early in `IResearchAnalyzerFeature::stop()` too?

Reply:
@jsteemann good point, I missed it.